cmd/kubelet/app/server.go

```diff
@@ -363,6 +364,7 @@ func UnsecuredDependencies(s *options.KubeletServer) (*kubelet.Dependencies, err
 	}
 
 	mounter := mount.New(s.ExperimentalMounterPath)
+	subpather := subpath.New()
```

Comment:
Not sure if it's possible, but can you pass mounter to subpather here (`subpath.New(mounter)`) instead of passing it in every `PrepareSafeSubpath` / `CleanSubPaths`?

Reply:
I can. Good idea.